cli/common.go

```diff
@@ -29,6 +29,21 @@ import (
 // fillCommonRuntime initializes common configuration settings, from several
 // input sources (in order: defaults, file config, environment overrides).
 func fillCommonRuntime() (*torcx.CommonConfig, error) {
+	// Determine OS release ID
+	OsRelease, err := torcx.CurrentOsVersionID(torcx.OsReleasePath)
+	if err != nil {
+		OsRelease = ""
+		logrus.WithFields(logrus.Fields{
+			"path": torcx.OsReleasePath,
+		}).Warn("unable to detect OS version-id")
+	}
+
+	return fillCommonRuntimeWithOsRelease(OsRelease)
+}
+
+// fillCommonRuntimeWithOsRelease is the same as fillCommonRuntime, but with
+// a user-specified OS release
+func fillCommonRuntimeWithOsRelease(OsRelease string) (*torcx.CommonConfig, error) {
```

Comment:
Just move this `OsRelease string` parameter to the original `fillCommonRuntime()` and let's just keep one of them.